src/constants/queries.ts

```diff
@@ -90,10 +91,15 @@ export const FlASHSYSTEM_STORAGECLASS_QUERIES = (
     queryItem: string
 ): string => {
 
+  const filterPVCWithoutLabel =`label_replace((kube_persistentvolume_claim_ref * on(persistentvolume) group_right(name) kube_persistentvolume_labels{label_odf_fs_storage_system=''}), "persistentvolumeclaim", "$1", "name", "(.+)")`
```

Comment:
Better to keep the consistency with other const, so consider changing to `filterPVWithoutLabel` (PV instead of PVC), as the label is related to the PV

Reply:
fixed